jf_agent/diagnostics.py

```diff
@@ -84,6 +84,44 @@ def capture_outdir_size(outdir):
     )
 
 
+def capture_download_data_summary(outdir):
+    # obtain file names from the directory
+    _, directories, filenames = next(os.walk(outdir))
+
+    # get the full file paths for each of the immediate
+    # subdirectories (we're assuming only a single level)
+    for directory in directories:
+        path = os.path.join(outdir, directory)
+        for file_name in os.listdir(path):
+            filenames.append(f'{directory}/{file_name}')
+
+    for filename in filenames:
+        if filename not in [
+            'jira_projects_and_versions.json',
+            'jira_issues.json',
+            'jira_fields.json',
+            'bb_commits.json',
+            'bb_projects.json',
+            'bb_prs.json',
+            'bb_repos.json',
+        ]:
+            filenames.remove(filename)
```

Comment:
lol, the first thing i learned about iterating, never remove an item from what you're iterating over 🤦‍♀️